src/main/java/com/kodilla/ecommercee/service/CartService.java

```diff
@@ -35,6 +39,26 @@ public Cart createCart() {
         return cartRepository.save(new Cart());
     }
 
+    public Cart saveCart(Cart cart) {
+        return cartRepository.save(cart);
+    }
+
+    public Cart getCart(Long cartId) throws CartNotFoundException {
+        if(cartRepository.getOne(cartId) == null ){
```

Comment:
Moim zdaniem getOne jest nieco niebezpieczne. Ono nie zwróci Ci null, jeżeli nic nie znajdzie, tylko wyrzuci wyjątek, który po stronie Postmana da niewiele mówiący błąd=500, a powinien wyskoczyć 404=not found. Dalsza część else nawet nie jest sprawdzana. Lepiej jest dać findById, bo da Ci typ optional i nie wyrzuci exception, ale wyrzuci Ci twój wyjątek, a całość skróci się do jednej lini